### DISCUSSION TITLE
Purchased a domain for whisper.cpp

### DISCUSSION THREAD

**Discussion by bobqianic** [ggml-org/whisper.cpp#1291] [General]
Yesterday, I purchased the domain [whispercpp.org](https://whispercpp.org), which can serve as the domain for the documentation site of whisper.cpp. I plan to use **Jupyter Book** to write the documentation and then publish it online using GitHub Pages. **Jupyter Book** is really convenient to use, as it supports markdown syntax and offers many features not supported by GitHub, like displaying mathematical formulas. I've set up a demo using this domain for now. Editing is also very straightforward; after finishing the markdown, we can use GitHub Action to automatically compile the documentation into HTML files and then publish them. @ggerganov

**Comment by bobqianic** [comment]
Could you take a look at this and let me know if it meets your expectations? I've always found using the wiki cumbersome, particularly because the sidebar isn't collapsible. If this approach satisfies you, I'll go ahead and renew the domain for a few more years, then transfer it to you. I also think it might be wise to create a separate repository just for the whisper.cpp documents. Granting write permissions to action-bot feels risky to me. @ggerganov

**Reply by ggerganov** [reply]
Thanks for this effort! I don't think I will have the capacity to maintain this, but if you are up to it, feel free to create it any way you like. I do think that having everything available here in the repo is better, but I wouldn't mind this extra documentation site to provide extra info.

**Comment by jaybinks** [comment]
Point the domain to a page hosted in the github repo.
Possibly even the readme.md ensures no confusion and makes its super easy to maintain.
